### PR TITLE
Fixing couchbase tickets drop 1

### PR DESCRIPTION
**Pull request by ghost**
Fixed the following tickets as part of this PR.
DOC-9105
DOC-8481
DOC-8480
DOC-8449
DOC-8082
DOC-8448
DOC-8096

### CODE REVIEW

File: modules/fts/pages/fts-monitor.adoc [hunk 1]
```diff
@@ -0,0 +1,144 @@
+= Monitor 
+
+== Monitor runtime queries
+
+Users of the search service might want to monitor the currently running active queries on the Full Text Search (FTS) search page. Monitoring helps get insights into slow queries or for other debugging purposes.
```
Comment (sreekanth-cb) [inline modules/fts/pages/fts-monitor.adoc]: Trailing "search page" is redundant here. 

Users of the search service might want to monitor the currently running active queries on the Full Text Search (FTS) service. looks enough?

File: modules/fts/pages/fts-monitor.adoc [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+image::fts-monitoring statistics1.png[,820,align=left] 
+
+The statistics display the following graphical charts:
+
+* Search Query Latency: The average time to run the search query.
+* Search Docs: The no. of documents scanned for the search query.
+* Search Disk Size: The total disk file size occupied by the fts index
+* Search Disk Files: The number of index files on the disk.
+* Search Memory Segments: The no. of memory segments in the full text search index across all partitions.
+* Search Disk Segments:  The no. of index segments across all partitions resident in the disk.
```
Comment (sreekanth-cb) [inline modules/fts/pages/fts-monitor.adoc]: on the disk Vs in the disk?  You have used them interchangeably?

File: modules/fts/pages/fts-monitor.adoc [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+A graphical display of the statistics is shown below:
+
+[#fts_fts_monitoring_statistics2]
+image::fts-monitoring statistics1.png[,820,align=left] 
+
+The statistics display the following graphical charts:
+
+* Search Query Latency: The average time to run the search query.
+* Search Docs: The no. of documents scanned for the search query.
```
Comment (sreekanth-cb) [inline modules/fts/pages/fts-monitor.adoc]: The no. of documents processed during the indexing.

File: modules/fts/pages/fts-monitor.adoc [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+* Search Disk Size: The total disk file size occupied by the fts index
```
Comment (sreekanth-cb) [inline modules/fts/pages/fts-monitor.adoc]: The total size of disk space occupied by the fts index

File: modules/fts/pages/fts-monitor.adoc (hunk 2 above)
Comment (sreekanth-cb) [reply modules/fts/pages/fts-monitor.adoc]: The no. of disk segments in the full text search index across all partitions.

File: modules/fts/pages/fts-monitor.adoc [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+* Search Disk Files: The number of index files on the disk.
+* Search Memory Segments: The no. of memory segments in the full text search index across all partitions.
+* Search Disk Segments:  The no. of index segments across all partitions resident in the disk.
+* Search Mutations Remaining: The no. of mutations not yet indexed.
+* Search Partitions: The no. of partitions scanned for the search query.
```
Comment (sreekanth-cb) [inline modules/fts/pages/fts-monitor.adoc]: The no. of actual index partitions.

File: modules/fts/pages/fts-monitor.adoc [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+* Search Partitions Expected: The no. of partitions expected to be scanned for the search query.
```
Comment (sreekanth-cb) [inline modules/fts/pages/fts-monitor.adoc]: The no. of index partitions expected as per the index definition.

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+== Impacts on the live traffic updates
+The live traffic is never functionally affected. Nevertheless, we can't entirely rule out the performance impacts of a concurrent rebalance on a live cluster.
+
+== Tips for faster rebalance operations
+As the rebalance is a resource-consuming cluster management operation, It is always recommended to perform rebalance during off-peak hours. Search service moves or builds the index partitions one at a time per node during the rebalance operations, which could significantly increase the overall time taken for the rebalance operation.
+
+There is a configurable option [maxConcurrentPartitionMovesPerNode] to bring the additional concurrency to how we move/build partitions during a rebalance operation. 
+Suppose we override this parameter (maxConcurrentPartitionMovesPerNode to N) as a runtime cluster option. In that case, we could concurrently build that many partitions in parallel per node at a time, and the rebalance ought to complete faster.
+
+*Configuring this `maxConcurrentPartitionMovesPerNode` in a cluster in CC*
```
Comment (sreekanth-cb) [inline modules/fts/pages/fts-rebalance-failover.adoc]: Change "in CC" => "on server release 7.0"?

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 8]
```diff
@@ -0,0 +1,52 @@
+= Rebalance/Failover for Search Service
+
+== Rebalance
```
Comment (sreekanth-cb) [inline modules/fts/pages/fts-rebalance-failover.adoc:3]: Did anyone from the docs team verified the language aspects of these contents? If not, please do for this PR.

File: modules/fts/pages/fts-monitor.adoc (hunk 1 above)
Comment (couchbase123) [reply modules/fts/pages/fts-monitor.adoc]: Proposal: Users of the full text search service might want to monitor the currently running active queries. Monitoring helps get insights into slow queries or for other debugging purposes.

File: modules/fts/pages/fts-monitor.adoc (hunk 2 above)
Comment (couchbase123) [reply modules/fts/pages/fts-monitor.adoc]: "on disk" in "Search Disk Files: The number of index files on disk"
Change no. to "number of" in all occurrences

For the search disk segments - change description to
 The number of disk segments in the full text search index across all resident disk partions.

File: modules/fts/pages/fts-monitor.adoc (hunk 3 above)
Comment (couchbase123) [reply modules/fts/pages/fts-monitor.adoc]: Change no. to "number of" in all occurrences

File: modules/fts/pages/fts-monitor.adoc (hunk 4 above)
Comment (couchbase123) [reply modules/fts/pages/fts-monitor.adoc]: Change no. to "number of" in all occurrences

The total size of disk space occupied by the full text search index

File: modules/fts/pages/fts-monitor.adoc (hunk 5 above)
Comment (couchbase123) [reply modules/fts/pages/fts-monitor.adoc]: Change no. to "number of" in all occurrences

File: modules/fts/pages/fts-monitor.adoc (hunk 6 above)
Comment (couchbase123) [reply modules/fts/pages/fts-monitor.adoc]: Change no. to "number of" in all occurrences

File: modules/ROOT/nav.adoc [hunk 9]
```diff
@@ -285,10 +285,13 @@
     ***** xref:n1ql:n1ql-language-reference/statistics-delete.adoc[Delete Statistics]
    **** xref:n1ql:n1ql-language-reference/upsert.adoc[UPSERT]
 * xref:fts:fts-introduction.adoc[Full Text Search (FTS)]
+
```
Comment (simon-dew) [inline modules/ROOT/nav.adoc:288]: Remove blank line

File: modules/fts/pages/fts-creating-index-from-UI.adoc [hunk 10]
```diff
@@ -8,53 +8,50 @@ Once you have the appropriate credentials, and have loaded the `travel-sample bu
...
+image::fts-fts-console-initial.png[,750,align=left]
 
 The console contains areas for the display of indexes and aliases: but both are empty since none has yet been created.
 
 == Add Index
 
-To create an index, left-click on the Add Index button, towards the right-hand side:
-
-image::fts-add-index-button.png[,150,align=left]
+To create an index, left-click on the *Add Index* button, towards the right-hand side
```
Comment (simon-dew) [inline modules/fts/pages/fts-creating-index-from-UI.adoc]: Full stop at end

```suggestion
To create an index, left-click on the *Add Index* button, towards the right-hand side.
```

File: modules/fts/pages/fts-introduction.adoc [hunk 11]
```diff
@@ -1,15 +1,16 @@
 = Introduction to Full Text Search
-:page-aliases: full-text-intro.adoc
```
Comment (simon-dew) [inline modules/fts/pages/fts-introduction.adoc:2]: Why has this page alias been removed? Page aliases are necessary to enable users to find equivalent information across versions of the product. Please reinstate, unless this same page alias is being used elsewhere.

File: modules/fts/pages/fts-introduction.adoc [hunk 12]
```diff
@@ -1,15 +1,16 @@
...
-
 
 [abstract]
 _Full Text Search_ (FTS) lets you create, manage, and query _indexes_, defined on JSON documents within a Couchbase bucket.
 
-== Full Text
+== Full Text Search
+
+Provided by the xref:services-and-indexes/services/search-service.adoc[Search Service], lets users create, manage, and query multi-purposed indexes defined on JSON documents within a Couchbase bucket.
```
Comment (simon-dew) [inline modules/fts/pages/fts-introduction.adoc]: As it is, this sentence doesn't seem to meet DOC-9105. Adding "full text search" for SEO and defining the abbreviation.

```suggestion
Provided by the xref:services-and-indexes/services/search-service.adoc[Search Service], full-text search (FTS) enables you to create, manage, and query multi-purposed indexes defined on JSON documents within a Couchbase bucket.
```

File: modules/fts/pages/fts-monitor.adoc [hunk 13]
```diff
@@ -0,0 +1,144 @@
...
+   from
+      ": 0, "timeout": 10000, "index": "DemoIndex2" }, "executionTime": "2.216451567s" }, "9": { "QueryContext": { "query": { "query": "german" }, "size": 10, "
+      from
+         ": 0, "timeout": 10000, "index": "DemoIndex" }, "executionTime": "13.863849125s" } 
+         
+        }
+    } 
+}
+
+----
```
Comment (simon-dew) [inline modules/fts/pages/fts-monitor.adoc]: More user-friendly markup for code blocks — keep input separate from output, add code highlighting

```suggestion
[source,shell]
---- 
curl -XGET -H "Content-Type: application/json" -u<UserName:Password> http://localhost:9200/api/query
----

.Sample Output
[source,json]
----
{ "status": "ok", "totalActiveQueryCount": 3, "filteredActiveQueries": { "queryCount": 3, "queryMap": { "10": { "QueryContext": { "query": { "query": "american" }, "size": 10, "
from
   ": 0, "timeout": 10000, "index": "DemoIndex1" }, "executionTime": "9.700851426s" }, "11": { "QueryContext": { "query": { "query": "russian" }, "size": 10, "
   from
      ": 0, "timeout": 10000, "index": "DemoIndex2" }, "executionTime": "2.216451567s" }, "9": { "QueryContext": { "query": { "query": "german" }, "size": 10, "
      from
         ": 0, "timeout": 10000, "index": "DemoIndex" }, "executionTime": "13.863849125s" } 
         
        }
    } 
}
----
```

File: modules/fts/pages/fts-monitor.adoc [hunk 14]
```diff
@@ -0,0 +1,144 @@
...
+    "status": "ok",
+    "totalActiveQueryCount": 0,
+    "filteredActiveQueries": {
+        "indexName": "DemoIndex",
+        "queryCount": 0,
+        "queryMap": {}
+    }
+}
+
+----
```
Comment (simon-dew) [inline modules/fts/pages/fts-monitor.adoc]: Keep input separate from output, add source highlighting

```suggestion
[source,shell]
----
curl -XGET -H "Content-Type: application/json" -uAdministrator:asdasd 'http://localhost:9200/api/query?longerThan=10s'
----

.Sample Output
[source,json]
----
{
    "status": "ok",
    "totalActiveQueryCount": 0,
    "filteredActiveQueries": {
        "indexName": "DemoIndex",
        "queryCount": 0,
        "queryMap": {}
    }
}
----
```

File: modules/fts/pages/fts-monitor.adoc [hunk 15]
```diff
@@ -0,0 +1,144 @@
...
+
+/api/query/index/{indexName}
```
Comment (simon-dew) [inline modules/fts/pages/fts-monitor.adoc:62]: AsciiDoctor interprets the path parameter as an attribute, which causes errors at build time. Wrap the endpoint in a listing block to escape it.

```suggestion
....
/api/query/index/{indexName}
....
```

File: modules/fts/pages/fts-monitor.adoc [hunk 16]
```diff
@@ -0,0 +1,144 @@
...
+
+The users can use the /api/query/index/{indexName} endpoint to get the details of all the active queries for any given FTS index in the system. With this endpoint, the users can also use the longerThan argument to filter the queries running beyond the given span of time.
```
Comment (simon-dew) [inline modules/fts/pages/fts-monitor.adoc]: Escape attribute

```suggestion
The users can use the `pass:c[/api/query/index/{indexName}]` endpoint to get the details of all the active queries for any given FTS index in the system. With this endpoint, the users can also use the longerThan argument to filter the queries running beyond the given span of time.
```

File: modules/fts/pages/fts-monitor.adoc [hunk 17]
```diff
@@ -0,0 +1,144 @@
...
+The longerThan duration string is a signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as "20s", "-1.5h" or "2h45m".
+Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+ 
+For example, 
+
+----
+
+curl -XGET -H "Content-Type: application/json" -uAdministrator:asdasd 'http://localhost:9200/api/query/index/<indexName>?longerThan=1ms
+
+Sample Output
```
Comment (simon-dew) [inline modules/fts/pages/fts-monitor.adoc]: Separate input and output, syntax highlighting

```suggestion
For example, 

[source,shell]
----
curl -XGET -H "Content-Type: application/json" -uAdministrator:asdasd 'http://localhost:9200/api/query/index/<indexName>?longerThan=1ms
----

.Sample Output
[source,json]
----
```

File: modules/fts/pages/fts-monitor.adoc [hunk 18]
```diff
@@ -0,0 +1,144 @@
...
+   ": 0, "timeout": 10000, "index": "DemoIndex" }, "executionTime": "17.340715297s" }, "5": { "QueryContext": { "query": { "query": "german" }, "size": 10, "
+   from
+      ": 0, "timeout": 10000, "index": "DemoIndex" }, "executionTime": "9.561917571s" }, "6": { "QueryContext": { "query": { "query": "pale ale" }, "size": 10, "
+      from
+         ": 0, "timeout": 10000, "index": "DemoIndex" }, "executionTime": "1.239720897s" } 
+        } 
+    } 
+}
+
+----
```
Comment (simon-dew) [inline modules/fts/pages/fts-monitor.adoc]:
```suggestion
----
```

File: modules/fts/pages/fts-query-string-syntax.adoc [hunk 19]
```diff
@@ -1,5 +1,4 @@
 = Query String Syntax
-:page-aliases: query-string-queries.adoc
```
Comment (simon-dew) [inline modules/fts/pages/fts-query-string-syntax.adoc]: Why has this page alias been removed? Page aliases are necessary to enable users to find equivalent information across versions of the product. Please reinstate, unless this same page alias is being used elsewhere.

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 20]
```diff
@@ -0,0 +1,52 @@
+= Rebalance/Failover for Search Service
+
+== Rebalance 
+The search service maintains a cluster-wide set of index definitions and metadata, which allows the redistribution of indexes and index replicas during a rebalance.
+
+== Handling partitions during a rebalance
```
Comment (simon-dew) [inline modules/fts/pages/fts-rebalance-failover.adoc:6]: Blank line after heading for readability

```suggestion
== Handling partitions during a rebalance

```

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 21]
```diff
@@ -0,0 +1,52 @@
...
+The search service redistributes the index partitions across the available search service nodes for a balanced partition-node assignment during a rebalance operation. The newly assigned index partitions are built afresh over DCP feed on the new nodes. And once the new partitions are built up to the current / latest sequence numbers, they are promoted to take the live traffic, and the older partitions are deleted from the system. 
+
+== Impacts on the live traffic updates
```
Comment (simon-dew) [inline modules/fts/pages/fts-rebalance-failover.adoc:10]: Blank line after heading for readability

```suggestion
== Impacts on the live traffic updates

```

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 22]
```diff
@@ -0,0 +1,52 @@
...
+The live traffic is never functionally affected. Nevertheless, we can't entirely rule out the performance impacts of a concurrent rebalance on a live cluster.
+
+== Tips for faster rebalance operations
```
Comment (simon-dew) [inline modules/fts/pages/fts-rebalance-failover.adoc:14]: Blank line after heading for readability

```suggestion
== Tips for faster rebalance operations

```

File: modules/fts/pages/fts-rebalance-failover.adoc (hunk 7 above)
Comment (simon-dew) [reply modules/fts/pages/fts-rebalance-failover.adoc]: I would just remove the version reference altogether. Also, if this is supposed to be a heading, use heading markup.

```suggestion
=== Configuring the maximum concurrent partition moves per node
```

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 23]
```diff
@@ -0,0 +1,52 @@
...
+
+*Configuring this `maxConcurrentPartitionMovesPerNode` in a cluster in CC*
+Use the update endpoint for manager options.
+
+[source,json]
+----
+
+curl -XPUT -uAdministrator:asdasd http://<nodeIP>:8094/api/managerOptions -d ' {| |"maxConcurrentPartitionMovesPerNode":"5"}'
+
+----
```
Comment (simon-dew) [inline modules/fts/pages/fts-rebalance-failover.adoc]: Correct source language

```suggestion
[source,shell]
----
curl -XPUT -uAdministrator:asdasd http://<nodeIP>:8094/api/managerOptions -d ' {| |"maxConcurrentPartitionMovesPerNode":"5"}'
----
```

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 24]
```diff
@@ -0,0 +1,52 @@
...
+
+Checking the current value for `maxConcurrentPartitionMovesPerNode` in a cluster
+
+[source,json]
+----
+
+curl -XGET -uAdministrator:asdasd http://<nodeIP>:8094/api/manager
+
+----
```
Comment (simon-dew) [inline modules/fts/pages/fts-rebalance-failover.adoc]:
```suggestion
[source,shell]
----
curl -XGET -uAdministrator:asdasd http://<nodeIP>:8094/api/manager
----
```

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 25]
```diff
@@ -0,0 +1,52 @@
...
+
+Note: When multiple partitions are built parallel, it needs more RAM and mandates a higher RAM quota. As the rebalance operations consume resources, it is always advisable to plan them during non-peak hours. One way to speed up the rebalance operation is to enable the movement of partitions parallelly in a configurable way.
+
+== Dealing with rebalance failures
+
+=== Failovers
```
Comment (simon-dew) [inline modules/fts/pages/fts-rebalance-failover.adoc:40]: Blank line after heading for readability

```suggestion
=== Failovers 

```

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 26]
```diff
@@ -0,0 +1,52 @@
...
+----
+
+Note: When multiple partitions are built parallel, it needs more RAM and mandates a higher RAM quota. As the rebalance operations consume resources, it is always advisable to plan them during non-peak hours. One way to speed up the rebalance operation is to enable the movement of partitions parallelly in a configurable way.
+
+== Dealing with rebalance failures
+
+=== Failovers 
+During failover of Search service nodes, there is no partition movement, and hence the failover-rebalance is instantaneous. Search service promotes the replica index partitions to primary so that those serve the live cluster traffic instantly. Users could use failover and recovery rebalance for applying patches or upgrading software/hardware for a shorter duration. 
+ 
+=== Things to keep track of during a rebalance
```
Comment (simon-dew) [inline modules/fts/pages/fts-rebalance-failover.adoc:44]: Blank line after heading for readability

```suggestion
=== Things to keep track of during a rebalance

```

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 27]
```diff
@@ -0,0 +1,52 @@
...
+During a strict recovery rebalance operation (no extra node additions/removals), the index partitions residing on the recovered node would be reused. And this ensures a quick recovery rebalance operation.
+
+=== Failover-recovery steps
```
Comment (simon-dew) [inline modules/fts/pages/fts-rebalance-failover.adoc:48]: Blank line after heading for readability

```suggestion
=== Failover-recovery steps

```

**Review by simon-dew — CHANGES_REQUESTED** [review]
Mostly this is great, but I have requested some markup changes for readability and improved UX. More importantly, I’m wondering why page aliases have been removed.

File: modules/fts/pages/fts-custom-filters.adoc [hunk 28]
```diff
@@ -4,7 +4,7 @@ Custom filters can be viewed and modified from the index’s configuration page
 
 == Add Custom Filter
 
-In order to add a custom filter to a Full Text Index via the Couchbase Capella UI, the following permissions are required:
+In order to add a custom filter to a Full Text Index via the Couchbase Cloud UI, the following permissions are required:
```
Comment (maria-robobug) [inline modules/fts/pages/fts-custom-filters.adoc]: Hmm, why are we removing mention of `Capella` here?

File: modules/fts/pages/fts-date-time-parsers.adoc [hunk 29]
```diff
@@ -7,7 +7,7 @@ The Search Service expects dates to be in the format specified by RFC-3339 , whi
 A date/time parser tells the Search Service ahead of time what the date/time layout of a field will be. If the Type of a child field is set to `datetime`, and the time layout(s) found in that field is not in RFC-3339 format, then you can specify a custom date/time parser that contains the layouts that the Search Service is to expect.
 
 NOTE: Permissions Required
-In order to add a custom date/time parser to a Full Text Index via the Couchbase Capella UI, the following permissions are required:
+In order to add a custom date/time parser to a Full Text Index via the Couchbase Cloud UI, the following permissions are required:
```
Comment (maria-robobug) [inline modules/fts/pages/fts-date-time-parsers.adoc]: Same here, shouldn't this be `Capella`?

File: modules/ROOT/nav.adoc (hunk 9 above)
Comment (simon-dew) [reply modules/ROOT/nav.adoc:288]:
```suggestion
```

File: modules/fts/pages/fts-introduction.adoc [hunk 30]
```diff
@@ -1,15 +1,16 @@
 = Introduction to Full Text Search
```
Comment (simon-dew) [inline modules/fts/pages/fts-introduction.adoc:1]:
```suggestion
= Introduction to Full Text Search
:page-aliases: full-text-intro.adoc
```

File: modules/fts/pages/fts-monitor.adoc (hunk 1 above)
Comment (simon-dew) [reply modules/fts/pages/fts-monitor.adoc]:
```suggestion
Users of the full text search service might want to monitor the currently running active queries.
Monitoring helps get insights into slow queries, or for other debugging purposes.
```

File: modules/fts/pages/fts-monitor.adoc [hunk 31]
```diff
@@ -0,0 +1,151 @@
+= Monitor 
+
+== Monitor runtime queries
+
+Users of the search service might want to monitor the currently running active queries on the Full Text Search (FTS) search page. Monitoring helps get insights into slow queries or for other debugging purposes.
+FTS provides new REST endpoints to supervise the runtime queries.
+
+/api/query
```
Comment (simon-dew) [inline modules/fts/pages/fts-monitor.adoc:10]:
```suggestion
....
/api/query
....
```

File: modules/fts/pages/fts-monitor.adoc [hunk 32]
```diff
@@ -0,0 +1,151 @@
...
+      ": 0, "timeout": 10000, "index": "DemoIndex2" }, "executionTime": "2.216451567s" }, "9": { "QueryContext": { "query": { "query": "german" }, "size": 10, "
+      from
+         ": 0, "timeout": 10000, "index": "DemoIndex" }, "executionTime": "13.863849125s" } 
+         
+        }
+    } 
+}
+----
+
+The api/query endpoint takes an optional argument longerThan. With this argument, the users can filter the queries running beyond the given span of time.
```
Comment (simon-dew) [inline modules/fts/pages/fts-monitor.adoc]:
```suggestion
The `api/query` endpoint takes an optional argument `longerThan`.
With this argument, the users can filter the queries running beyond the given span of time.
```

File: modules/fts/pages/fts-monitor.adoc (hunk 3 above)
Comment (simon-dew) [reply modules/fts/pages/fts-monitor.adoc]:
```suggestion
* Search Docs: The number of documents scanned for the search query.
```

File: modules/fts/pages/fts-monitor.adoc (hunk 4 above)
Comment (simon-dew) [reply modules/fts/pages/fts-monitor.adoc]:
```suggestion
* Search Disk Size: The total size of disk space occupied by the fts index.
```

File: modules/fts/pages/fts-monitor.adoc (hunk 2 above)
Comment (simon-dew) [reply modules/fts/pages/fts-monitor.adoc]:
```suggestion
* Search Disk Segments: The number of disk segments in the full text search index across all resident disk partions.
```

File: modules/fts/pages/fts-monitor.adoc [hunk 33]
```diff
@@ -0,0 +1,151 @@
...
+[#fts_fts_monitoring_statistics2]
+image::fts-monitoring statistics1.png[,820,align=left] 
+
+The statistics display the following graphical charts:
+
+* Search Query Latency: The average time to run the search query.
+* Search Docs: The no. of documents scanned for the search query.
+* Search Disk Size: The total disk file size occupied by the fts index
+* Search Disk Files: The number of index files on the disk.
+* Search Memory Segments: The no. of memory segments in the full text search index across all partitions.
```
Comment (simon-dew) [inline modules/fts/pages/fts-monitor.adoc]:
```suggestion
* Search Memory Segments: The number of memory segments in the full text search index across all partitions.
```

File: modules/fts/pages/fts-monitor.adoc (hunk 5 above)
Comment (simon-dew) [reply modules/fts/pages/fts-monitor.adoc]:
```suggestion
* Search Partitions: The number of actual index partitions.
```

File: modules/fts/pages/fts-monitor.adoc [hunk 34]
```diff
@@ -0,0 +1,151 @@
...
+* Search Disk Segments:  The no. of index segments across all partitions resident in the disk.
+* Search Mutations Remaining: The no. of mutations not yet indexed.
```
Comment (simon-dew) [inline modules/fts/pages/fts-monitor.adoc]:
```suggestion
* Search Mutations Remaining: The number of mutations not yet indexed.
```

File: modules/fts/pages/fts-monitor.adoc (hunk 6 above)
Comment (simon-dew) [reply modules/fts/pages/fts-monitor.adoc]:
```suggestion
* Search Partitions Expected: The number of partitions expected to be scanned for the search query.
```

File: modules/fts/pages/fts-monitor.adoc [hunk 35]
```diff
@@ -0,0 +1,151 @@
...
+1. In the left pane, click *Dashboard*.
+2. On the Dashboard page, click the *Search indexes* drop-down list and select the full text search index you want to view the statistics.
+
+A graphical display of the statistics is as shown below:
+
+[#fts_fts_monitoring_statistics1]
+image::fts-monitoring statistics1.png[,820,align=left]
+
+The statistics display the following graphical charts for full text search index:
+* Search Mutations Remaining: The no. of mutations not yet indexed.
```
Comment (simon-dew) [inline modules/fts/pages/fts-monitor.adoc]:
```suggestion
* Search Mutations Remaining: The number of mutations not yet indexed.
```

File: modules/fts/pages/fts-monitor.adoc [hunk 36]
```diff
@@ -0,0 +1,151 @@
...
+* Search Partitions Expected: The no. of partitions expected to be scanned for the search query.
+* Search Records to Persist: The no. of index records not yet persisted to the disk.
+* Search Index Rate: The rate (In Bytes per second) of indexing plain text.
+* Search Result Rate: The rate (In Bytes per second) of returning the search result.
+* Search Compaction Rate: The rate (In Bytes per second) of compaction of the index segments.
+* Search Query Rate: The no. of queries run per second.
+* Search Query Error Rate: The no. of queries per second (Including timeouts) that resulted in an error.
+* Search Slow Queries: The no. of queries that run slowly (greater than 5 seconds to run) ) per second.
+* Search Query Timeout Rate: The no. of queries that timeout per second.
+* Term Searchers Start Rate: The no. of term searchers started per second.
```
Comment (simon-dew) [inline modules/fts/pages/fts-monitor.adoc]:
```suggestion
* Search Records to Persist: The number of index records not yet persisted to the disk.
* Search Index Rate: The rate (In Bytes per second) of indexing plain text.
* Search Result Rate: The rate (In Bytes per second) of returning the search result.
* Search Compaction Rate: The rate (In Bytes per second) of compaction of the index segments.
* Search Query Rate: The number of queries run per second.
* Search Query Error Rate: The number of queries per second (Including timeouts) that resulted in an error.
* Search Slow Queries: The number of queries that run slowly (greater than 5 seconds to run) ) per second.
* Search Query Timeout Rate: The number of queries that timeout per second.
* Term Searchers Start Rate: The number of term searchers started per second.
```

File: modules/fts/pages/fts-custom-filters.adoc (hunk 28 above)
Comment (simon-dew) [reply modules/fts/pages/fts-custom-filters.adoc]:
```suggestion
In order to add a custom filter to a Full Text Index via the Couchbase Capella UI, the following permissions are required:
```

File: modules/fts/pages/fts-date-time-parsers.adoc (hunk 29 above)
Comment (simon-dew) [reply modules/fts/pages/fts-date-time-parsers.adoc]:
```suggestion
In order to add a custom date/time parser to a Full Text Index via the Couchbase Capella UI, the following permissions are required:
```

**Review by simon-dew — CHANGES_REQUESTED** [review]
Further review changes

File: modules/fts/pages/fts-query-string-syntax.adoc [hunk 37]
```diff
@@ -1,5 +1,4 @@
 = Query String Syntax
```
Comment (simon-dew) [inline modules/fts/pages/fts-query-string-syntax.adoc]:
```suggestion
= Query String Syntax
:page-aliases: query-string-queries.adoc
```

**Review by simon-dew — CHANGES_REQUESTED** [review]
Final change

**Review by simon-dew — APPROVED** [review]

**Pull request merged**